### ISSUE TITLE
Update doc/tutorial_regression.rst

### ISSUE THREAD

**Issue by filipschlembach** [scikit-learn-contrib/MAPIE#120]
__Is your feature request related to a problem? Please describe.__
Running the tutorial [3. Estimating the uncertainty with different sklearn-compatible regressors](https://mapie.readthedocs.io/en/latest/tutorial_regression.html#estimating-the-uncertainty-with-different-sklearn-compatible-regressors) produces the error `No valid loss function found. You must provide a loss function to train. [...]` when fitting the Keras mlp.
It also produces the future warning ` UserWarning: ``build_fn`` will be renamed to ``model`` in a future release, at which point use of ``build_fn`` will raise an Error instead.` from `scikeras.wrappers import KerasRegressor`.

__Describe the solution you'd like__
The issue can be solved by changing
```
mlp_model = KerasRegressor(
    build_fn=mlp,
    epochs=500,
    verbose=0
)
```
to
```
mlp_model = KerasRegressor(
    model=mlp,
    epochs=500,
    verbose=0,
    loss='mse'
)
```
